src/testing.c

```diff
@@ -108,16 +111,158 @@ void testing(){
     //Free input arrays
     free_8_16_64(input);
 
+    // Call 1.2. Comparison
+    //perf_test();
+}
 
+int perf_test(){
     //Create Random Numbers for performance test of array
     //8 | 32  | 64 | 128 | .... etc
-    printf("\n\n");printf(COLOR);printf("+-------------------------------------------+\n");
-    printf(COLOR);printf("| Randomized Arrays for performance testing |\n");
-    printf(COLOR);printf("+-------------------------------------------+\n");printf(RESET);
+    printf("\n\n");
+    printf(COLOR);printf("+--------------------------------+\n");
+    printf(COLOR);printf("| Arrays for performance testing |\n");
+    printf(COLOR);printf("+--------------------------------+\n");
+    printf(RESET);
+    printf("\nAll values in ticks\n\n");
+    
+    int **input_pt = (int **)malloc(7 * sizeof(int*)); 
+
+    if (randomizer_pt(input_pt) != 0) {
+        perror("ERROR: Problem in Random Number generation");
+        return 1;
+    }
 
-    int **input_pt = (int **)malloc(7 * sizeof(int*));
+    //show_randomizer_pt(input_pt, 20);
+    printf(COLOR);printf("Randomized\n");printf(RESET);
+    run_algos(input_pt);
```

Comment:
hier wird dann die Unterfunktion run algos aufgerufen mit dem Doppelpointer als Argument (in der Folge 3x)